session/session.go

```diff
@@ -0,0 +1,144 @@
+package session
```

Comment:
"session: In computer science, in particular networking, a session is a temporary and interactive information **interchange** between two or more communicating devices, or between a computer and user."

Session in this case(most of the cases) are not suitable for a Go package name. You need to select a name that describes the whole package, not the thing you only interested about. 
For example: 

This package handles:
- apple login cookie data
- communicates with our portal service
- returns a session info that is suitable for fastlane and comes from our service

So I would go with <the thing I am requesting from>.<what I request>: `devportalservice.SessionData()` client-side clarification, so this would mean a package name of `devportalservice` and GetSession fn to be renamed to `SessionData`.

Reply:
Fixed in:
https://github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/pull/61/commits/96d945ed090daf9b2988faba7e1b5ad0316997a9